src/java/org/apache/cassandra/service/ActiveRepairService.java

```diff
@@ -198,10 +198,10 @@ public static ThreadPoolExecutor repairCommandExecutor()
     private final Gossiper gossiper;
     private final Cache<Integer, Pair<ParentRepairStatus, List<String>>> repairStatusByCmd;
 
-    private final DebuggableThreadPoolExecutor clearSnapshotExecutor = DebuggableThreadPoolExecutor.createWithMaximumPoolSize("RepairClearSnapshot",
```

Comment:
I renamed this executor as it is not only clearing snapshots anymore. Also I had to make it public to call it from outside, obviously.

Reply:
given this is a `DebuggableThreadPoolExecutor` the name would have been exposed via JMX, so renaming the string could be a regression if we released that name.

Reply:
@dcapwell where are they exposed? Are not only `JMXEnabledThreadPoolExecutor`s exposed to a user?

Reply:
yes you are right, `JMXEnabledThreadPoolExecutor` exposes but this doesn't.